views/home/index.ejs

```diff
@@ -86,8 +85,47 @@
               >
                 Generate
               </button>
+              <!-- Button to trigger the help modal -->
+              <button

```

Comment:
I also don't know how I feel about it being transparent and then being solid when you hover over it. Could we try just making it a solid color like the other buttons?

Reply:
okay, i'll look into this tn!

Reply:
@sderacy what do you think? the lighter grey is after you hover over it, but originally, its just white and black.
<img width="100" alt="Screen Shot 2022-11-28 at 2 51 48 PM" src="https://user-images.githubusercontent.com/57613959/204368247-48d74ce2-f228-4167-898e-e7485a113c17.png">
<img width="81" alt="Screen Shot 2022-11-28 at 2 52 02 PM" src="https://user-images.githubusercontent.com/57613959/204368248-a1c646f4-b8c4-4f68-94d1-6de4701fc0da.png">

Reply:
this is def good accessibility wise - according to the color guidelines site.

Reply:
its also bootstrap colors which abide by the accessibility color guidelines

Reply:
It looks good on the dark version but as long as it looks fine on light mode, I think it looks dope.